### DISCUSSION TITLE
Things that should be added in osu! lazer

### DISCUSSION THREAD

**Discussion by PokedAPie** [ppy/osu#18933] [Ideas] [answered]
HELLO WONDERFUL DEVELOPERS, my name is Piedpie PokedaPie, i just have this idea going around in my head for days so why not suggest it? The following is here:
1. Smokes
- The smokes are an og to be tru, they make the game look more fun than you expected, anyways, heres the idea of smoking on each gamemode (osu! Std, Mania, Catch and Taiko)
Osu! Std - Same as Osu (Smoke then move cursor)
Osu! Mania - Same as Osu! Std
Osu! Catch - Same as Osu! Std
Osu! Taiko - Same as Osu! Std

2. Bug fixes
- Ik Ik, theres been alot of bugs lately on this alzer game, but i think suggesting this might improve the game a lil bit:

1. Make Osu! Lazer accesible on all devices
- This should be good as of now i can't play osu lazer on my huawei touchscreen tablet, it just crashed even though it had access on internal storage

2. Flip the screen landscape so the circle looks bigger

3. Tap the Screen like Osu! Droid (More Fingers)

4. Fix FPS drop a.k.a. lag spike

5. Ability to tranfer files on normal osu

6. Ability to use skin on osu! lazer

7. Make editing alot more simpler (for begginers)

8. Ability to post pp plays on osu! website

9. Ability to see what rank your score is on the leaderboards

10. The ability to find specific maps with photo using links

11. Add a variety of finding a specific map (like Tech, Stream, Jump, Control and etc)

12. The Ability to save a replay even when died on a map (losing but still able to save a replay)

**Comment by peppy** [comment, marked as answer]
thanks for your support. most of what you ask for is already planned, so stay tuned for the future.

**Reply by PokedAPie** [reply]
Ty Peppy
